File: sources/LiquidSense Release/me/aquavit/liquidsense/command/commands/BindsCommand.java
package me.aquavit.liquidsense.command.commands;

import me.aquavit.liquidsense.LiquidSense;
import me.aquavit.liquidsense.command.Command;
import me.aquavit.liquidsense.utils.client.ClientUtils;
import org.lwjgl.input.Keyboard;

public class BindsCommand extends Command {
    public BindsCommand() {
        super("binds");
    }

    @Override
    public void execute(final String[] args) {
        if (args.length > 1) {
            final String lowerCase = args[1].toLowerCase();
            if ("clear".equals(lowerCase)) {
                LiquidSense.moduleManager.getModules().forEach(module -> module.setKeyBind(0));
                chat("Removed all binds.");
                return;
            }
        }
        chat("§c§lBinds");
        LiquidSense.moduleManager.getModules().stream().filter(module -> module.getKeyBind() != 0).forEach(
                module -> ClientUtils.displayChatMessage("§6> §c" + module.getName() + ": §a§l" + Keyboard.getKeyName(module.getKeyBind())));
        chatSyntax(".binds clear");
    }
}

File: sources/Kevin Reborn/kevin/command/commands/BindsCommand.kt
/*
 * This program is free software: you can redistribute it and/or modify
 *   it under the terms of the GNU General Public License as published by
 *   the Free Software Foundation, either version 3 of the License, or
 *   (at your option) any later version.
 *
 *   This program is distributed in the hope that it will be useful,
 *   but WITHOUT ANY WARRANTY; without even the implied warranty of
 *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *   GNU General Public License for more details.
 *
 *   You should have received a copy of the GNU General Public License
 *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
package kevin.command.commands

import kevin.command.ICommand
import kevin.main.KevinClient
import kevin.utils.ChatUtils
import org.lwjgl.input.Keyboard

class BindsCommand : ICommand {
    override fun run(args: Array<out String>?) {
        if (args != null && args.isNotEmpty()){
            if (args[0].equals("clear",true)){
                for (module in KevinClient.moduleManager.getModules()) module.keyBind = Keyboard.KEY_NONE
                ChatUtils.messageWithStart("§9Removed All Binds!")
                return
            }
        }
        ChatUtils.messageWithStart("§9Binds:")
        KevinClient.moduleManager.getModules().filter { it.keyBind != Keyboard.KEY_NONE }.forEach {
            ChatUtils.messageWithStart("§b> §9${it.name}: §a§l${Keyboard.getKeyName(it.keyBind)}")
        }
    }
}